src/step_definitions/performance.js

```diff
@@ -0,0 +1,49 @@
+import { defineSupportCode } from 'cucumber';
+import config from '../helpers/config.helper';
+
+const browsermob = require('browsermob-proxy').Proxy;
+const fs = require('fs');
+let proxy;
+
+defineSupportCode(function ({ When }) {
+  When(/^I start performance monitor mode$/, function () {
+    proxy = new browsermob({
+      port: config.browserMob.serverPort
+    });
+
+    let proxyReady = false;
+    proxy.start(config.browserMob.port, (err, data) => {
+      if (!err) {
+        proxy.startHAR(config.browserMob.port, 'test', true, true, () => {
```

Comment:
Do you know what is the purpose of 'test' parameter here ?

Reply:
It just a name:


`startHAR: function(port, name, captureHeaders, captureContent, captureBinaryContent, cb) {
        var _this = this
        ;

        cb = cb || arguments[arguments.length - 1];
        if (typeof name !== 'string')
            name = 'Page 1';

        var postData = 'initialPageRef=' + name;
        if (typeof captureHeaders === 'boolean') {
            postData += '&captureHeaders=' + captureHeaders.toString();
        }
        if (typeof captureContent === 'boolean') {
            postData += '&captureContent=' + captureContent.toString();
        }
        if (typeof captureBinaryContent === 'boolean') {
            postData += '&captureBinaryContent=' + captureBinaryContent.toString();
        }

        this.doReq('PUT', '/proxy/' + port + '/har', postData,
            // Check if we need to add in limits
            function(err, data) {
                var limit = false, limitObj = {};

                if (err) {
                    cb(err);
                } else {
                    ['downstreamKbps', 'upstreamKbps', 'latency'].forEach(function(key) {
                        if (_this[key]) {
                            limitObj[key] = _this[key];
                            limit = true;
                        }
                    });
                    if (limit) {
                        _this.limit(port, limitObj, cb);
                    } else {
                        cb(null);
                    }
                }
            }
        );
    },`